lib/charms/operator_libs_linux/v1/snap.py

```diff
@@ -907,8 +908,10 @@ def install_local(
     if dangerous:
         _cmd.append("--dangerous")
     try:
-        result = subprocess.check_output(_cmd, universal_newlines=True).splitlines()[0]
+        ansi_filter = re.compile(r"\x1B(?:[@-Z\\-_]|\[[0-?]*[ -/]*[@-~])")
```

Comment:
I'd make this a global (or a class attr) so as to only compile it once per run/import

Reply:
Made global in https://github.com/canonical/operator-libs-linux/pull/56/commits/8b13e63cdd98108dfd70e978ad4db8a225985b94.